### DISCUSSION TITLE
夜间模式下给评论区展开的当前页码加个蓝色

### DISCUSSION THREAD

**Discussion by cmmdwl** [the1812/Bilibili-Evolved#1605] [功能建议 / Ideas]
纯白的字体加粗和没加粗的看不清，要是能把展开后在哪页加个颜色就方便了

![image](https://user-images.githubusercontent.com/21263352/109326788-90831480-7892-11eb-9e4a-e7b5fee9672c.png)
夜间模式下下显示器加粗显示效果极差

**Comment by Geeyun-JY3** [comment]
是 bug，漏做了，这里应该跟随主题色的
